src/Symfony/Component/Serializer/Tests/Encoder/JsonEncodeTest.php

```diff
@@ -17,7 +17,7 @@
 
 class JsonEncodeTest extends TestCase
 {
-    private $encoder;
+    private $encode;
```

Comment:
hmm, altough I find the name weird, the tested class is JsonEncode, not JsonEncoder